jobs/openshift/cluster/integreatly/upgrade/Jenkinsfile

```diff
@@ -0,0 +1,231 @@
+
+def constants = [:]
+
+def prepareConstants(constants){
+  constants.TEST_SUITE_REPO = 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'  
+  constants.CUSTOMER_ADMIN_USERNAME = 'customer-admin' 
+  constants.CUSTOMER_ADMIN_PASSWORD = 'Password1'
+  constants.ADMIN_USERNAME = 'admin01'
+  constants.ADMIN_PASSWORD = 'Password1'
+}
+
+def ocLoginAsCustomerAdmin(constants) {
+    try {
+        String loginOutput = sh (
+            script: "oc login https://api.${params.clusterName}.openshift.com -u ${constants.CUSTOMER_ADMIN_USERNAME} -p ${constants.CUSTOMER_ADMIN_PASSWORD}",
+            returnStdout: true
+        ).trim()
+
+        println loginOutput
+        return loginOutput.contains("Login successful")
+    } catch (error) {
+        println "Login was not successful - ${error}"
+        return false
+    }
+}
+
+def prepareTestParams(constants) {
+  def jobParams = [
+    [$class: 'StringParameterValue', name: 'REPOSITORY', value: constants.TEST_SUITE_REPO],
+    [$class: 'StringParameterValue', name: 'BRANCH', value: params.testBranch],
+    [$class: 'StringParameterValue', name: 'CLUSTER_URL', value: "https://console.${params.clusterName}.openshift.com"], 
+    [$class: 'StringParameterValue', name: 'EVALS_USERNAME', value: "${params.userName}01"], 
+    [$class: 'StringParameterValue', name: 'ADMIN_USERNAME', value: constants.ADMIN_USERNAME],
+    [$class: 'StringParameterValue', name: 'ADMIN_PASSWORD', value: constants.ADMIN_PASSWORD], 
+    [$class: 'StringParameterValue', name: 'CUSTOMER_ADMIN_USERNAME', value: constants.CUSTOMER_ADMIN_USERNAME],
+    [$class: 'StringParameterValue', name: 'CUSTOMER_ADMIN_PASSWORD', value: constants.CUSTOMER_ADMIN_PASSWORD],
+    [$class: 'StringParameterValue', name: 'TIMEOUT_THRESHOLD', value: '2'],
```

Comment:
I think this should be 1 by default (or were you not able to execute it in this case?) and/or it could be parameterized.

Reply:
copied it from openshift-cluster-integreatly-test pipeline, it's hardcoded there too, but if you insist i can change it